CRM/Core/BAO/CustomField.php

```diff
@@ -562,6 +562,9 @@ public static function getKeyID($key, $all = FALSE) {
    * @throws CRM_Core_Exception
    */
   public static function getFieldObject($fieldID) {
+    if (empty(self::getField($fieldID))) {
+      return;
+    }
     $field = new CRM_Core_BAO_CustomField();
     $field->copyValues(self::getField($fieldID));
```

Comment:
I'd change this to a variable so the getField function doesn't have to be called twice.